tests/rewrites/i20002.scala

```diff
@@ -0,0 +1,38 @@
+object Reactions {
+  def main: Unit = {
+    Reactions += {
+      case 0 =>
+      case 1 =>
+    }
+
+    def bar: Int = ???
+
+    bar match {
+      case 0 =>
+      case 1 =>
+    }
+
+    def partPartial(i: Int): PartialFunction[Int, Unit] = {
+      case `i` =>
+    }
+
+    Reactions += {
+      val pp1 = partPartial(1)
+      val pp2 = partPartial(2)
+      def codeBlock = {
+        ???
+        ???
+      }
+      pp1 orElse pp2
+    }
+
+    val partialFunction = partPartial(1) orElse partPartial(2)
+    Reactions += {
+      partialFunction
+    }
```

Comment:
Technically, this could be rewritten to remove the braces (we would have to add a condition on the match clause that blocks contained elements other than the return - and update the flag that determines if we need to add a colon), however, given that these braces are optional under the old syntax, I chose to leave these ones in.
If it would be preferred that this reduces, I can draft that up next week.

Reply:
We cannot rewrite this to the following:
```scala
 Reactions += :
      partialFunction
```

but we can rewrite it to `Reactions += partialFunction`. Which is out of the scope of this PR and the ticket and should be left for tooling to do.